### PR TITLE
Add Oracle Explorer integrations to the web UI

### PR DESCRIPTION
**Pull request by user411**
Updates UI and adds some Oracle Explorer integrations

This includes code to format outcomes that @rorp added in #9 

This does not yet do attestations, but can create announcements from local events on an Oracle Explorer.

This requires two more fields on oracleServer 'getevent' returns in https://github.com/bitcoin-s/bitcoin-s/pull/3673

Screen Shot 2021-09-17 at 10 43 54 AM
Screen Shot 2021-09-17 at 10 44 12 AM
Screen Shot 2021-09-17 at 10 44 20 AM

### CODE REVIEW

File: oracle-server-ui/package.json [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
+  "version": "0.0.3",
   "scripts": {
     "ng": "ng",
     "start": "ng serve",
-    "startproxy": "ng serve --proxy-config proxy.conf.json",
+    "startproxy": "ng serve --watch --proxy-config proxy.conf.json",
     "build": "ng build",
     "watch": "ng build --watch --configuration development",
+    "watchproxy": "ng build --watch --configuration development --proxy-config proxy.conf.json",
     "test": "ng test"
```
Comment (Christewart) [inline oracle-server-ui/package.json:11]: Do these tests pass for you? This is what i get when i try to run

Screen Shot 2021-09-18 at 12 39 08 PM

Screen Shot 2021-09-18 at 12 40 09 PM

Comment (user411) [reply oracle-server-ui/package.json:11]: I have the same experience on my local. There are no meaningful tests currently, so I have only changed what the CI complains about.

**Comment by Christewart** [comment]
So i'm running a97ae6a and i'm trying to create an enumerated event but it can't. There is no error logs being printed to stdout (is there another logging location?). I think it is has to do with the fact that i don't have my oracle name filled in as indicated in the image. If this is the case, perhaps we should gray out or disable the create event box and give the users an error message say "you must initialize your oracle" or something like that

Screen Shot 2021-09-18 at 4 58 05 PM

**Comment by Christewart** [comment]
Might be nice to have a date picker for maturation time, for some reason i thought we already had this 

Screen Shot 2021-09-19 at 3 32 28 PM

**Comment by Christewart** [comment]
May want to disable the create event button if the oracle name is not set. I believe you cannot create the event to the button being disabled on the create event screen, but there isn't any message displayed as to why you can't create the event

Screen Shot 2021-09-19 at 3 34 48 PM

It wasn't entirely clear to me why  i couldn't create the event

Screen Shot 2021-09-19 at 3 36 01 PM

**Comment by Christewart** [comment]
Actually i can't seem to set my oracle name at all. When i click into the textbox and type my oracle name, and then click out of the textbox, nothing seems to happen. If i refresh the page the oracle name is gone 

Screen Shot 2021-09-19 at 3 37 52 PM

**Comment by user411** [comment]
I have no idea what version of the app you are running with the 'Digit Decomp' event still visible, but it's not top-of-tree.

Highly recommend running browser with debug window open set to not cache.

**Comment by user411** [comment]
The README.md was updated with my last checkini. The proxy server needs to run between the UI and the oracleServer for development and shipping at this point.

**Comment by user411** [comment]
Added
Screen Shot 2021-09-20 at 10 18 59 AM

**Comment by user411** [comment]
Updated
Screen Shot 2021-09-21 at 12 25 08 PM
)

**Pull request merged**